src/clover/__main__.py

```diff
@@ -1053,11 +1053,29 @@ def main(  # pylint: disable=too-many-locals, too-many-statements
             )
 
             if parsed_args.analyse:
+<<<<<<< HEAD
+                # Generate and save the various plots.
+                analysis.plot_outputs(  # type: ignore
+                    grid_times[scenario.grid_types],
+                    grid_profiles, #to check that
+                    initial_cw_hourly_loads,
+                    initial_electric_hourly_loads,
+                    initial_hw_hourly_loads,
+                    simulation.end_year - simulation.start_year,
+                    simulation_output_directory,
+                    output,
+                    simulation_number,
+                    system_performance_outputs,
+                    total_loads,
+                    total_solar_data[solar.SolarDataType.ELECTRICITY.value]
+                    * minigrid.pv_panel.pv_unit,
```

Comment:
In doing so, this has caused a discrepancy with the `dev` branch to open up as well here... Once fixed, this should hopefully go away 😄